src/dmd/mars.d

```diff
@@ -1345,6 +1362,70 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
         return true;
     }
 
+    /**
+     * Trigger an error messsage about an invalid switch.
```

Comment:
An error is not being "triggered", it is being printed. The trigger was where the decision to call this function lay.